packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.html

```diff
@@ -16,41 +16,73 @@
               <div onchange={handleKeyUp} class="slds-size_1-of-1">
-                <lightning-input class="facet__searchbox-input slds-p-horizontal_x-small slds-p-bottom_x-small" name={labels.search} type="search" label="facet-input" variant="label-hidden" placeholder={labels.search}></lightning-input>
+                <lightning-input
+                  class="facet__searchbox-input slds-p-horizontal_x-small slds-p-bottom_x-small"
+                  name={labels.search}
+                  type="search"
+                  label="facet-input"
+                  variant="label-hidden"
+                  placeholder={labels.search}
+                ></lightning-input>
               </div>
             </template>
 
             <template if:false={isFacetSearchActive}>
-              <template for:each={values} for:item="v">
-                <c-quantic-facet-value 
-                  onselectvalue={onSelectValue} 
-                  key={v.value} 
-                  item={v} 
-                  is-checked={v.checked}
-                  data-cy={v.value}
-                  display-as-link={isDisplayAsLink}>
-                </c-quantic-facet-value>
-              </template>
+              <fieldset>
+                <legend class="slds-assistive-text">{field}</legend>
+                <ul>
+                  <template for:each={values} for:item="v">
+                    <li class="slds-grid" key={v.value}>
+                      <c-quantic-facet-value
+                        onselectvalue={onSelectValue}
+                        item={v}
+                        key={v.value}
+                        is-checked={v.checked}
+                        data-cy={v.value}
+                        display-as-link={isDisplayAsLink}
+                      >
+                      </c-quantic-facet-value>
+                    </li>
+                  </template>
+                </ul>
+              </fieldset>
               <div class="facet__show slds-m-top_x-small slds-p-horizontal_x-small">
                 <template if:true={canShowLess}>
-                  <button variant="base" label={labels.showLess}
-                    class="facet__show-less slds-p-around_none" onclick={showLess} data-cy="less">
+                  <button
+                    variant="base"
+                    class="facet__show-less slds-p-around_none"
+                    aria-label="Show less"
+                    onclick={showLess}
+                    data-cy="less"
+                  >
                     <span class="facet__show-less_text">{labels.showLess}</span>
                   </button>
                 </template>
                 <template if:true={canShowMore}>
-                  <button variant="base" label={labels.showMore}
-                    class="facet__show-more slds-p-around_none" onclick={showMore} data-cy="more">
+                  <button
+                    variant="base"
+                    aria-label="Show more"
```

Comment:
You don't need that aria-label since you have a span with a label inside the button's HTML